server/src/test/java/org/elasticsearch/env/NodeEnvironmentTests.java

```diff
@@ -603,4 +678,62 @@ public NodeEnvironment newNodeEnvironment(String[] dataPaths, String sharedDataP
             .build();
         return new NodeEnvironment(build, TestEnvironment.newEnvironment(build));
     }
+
+    private static void overrideOldestIndexVersion(Version oldVersion, Path... dataPaths) throws IOException {
+        for (final Path dataPath : dataPaths) {
+            final Path indexPath = dataPath.resolve(METADATA_DIRECTORY_NAME);
+            if (Files.exists(indexPath)) {
+                try (DirectoryReader reader = DirectoryReader.open(new NIOFSDirectory(dataPath.resolve(METADATA_DIRECTORY_NAME)))) {
+                    final Map<String, String> userData = reader.getIndexCommit().getUserData();
+
+                    final IndexWriterConfig indexWriterConfig = new IndexWriterConfig(new KeywordAnalyzer());
+                    indexWriterConfig.setOpenMode(IndexWriterConfig.OpenMode.APPEND);
+                    indexWriterConfig.setCommitOnClose(false);
+                    indexWriterConfig.setRAMBufferSizeMB(1.0);
+                    indexWriterConfig.setMergeScheduler(new SerialMergeScheduler());
```

Comment:
I think the defaults are fine for our purposes here, no need to adjust them like this.